src/Component/ApiConsoleControl.php

```diff
@@ -80,6 +81,12 @@ protected function createComponentConsoleForm(): Form
 
         $form->addCheckbox('send_session_id', 'Send session id cookie');
 
+        $form->addTextArea('custom_headers', 'Custom headers')
+            ->setOption('description', Html::el()->setHtml('Each header on new line. For example: <code>User-agent: Mozilla/5.0</code>'));
```

Comment:
What about add there default value like ‘nette api console’

Reply:
User agent example? Whatever. Or default value of text area?

Reply:
i mean default value for text area but it is ok as it is